Examples/Tests/reduced_diags/analysis_reduced_diags_loadbalancecosts.py

```diff
@@ -29,18 +29,20 @@
 
 # From data header, data layout is:
 #     [step, time,
-#      cost_box_0, proc_box_0, lev_box_0, i_low_box_0, j_low_box_0, k_low_box_0,
-#      cost_box_1, proc_box_1, lev_box_1, i_low_box_1, j_low_box_1, k_low_box_1,
+#      cost_box_0, proc_box_0, lev_box_0, i_low_box_0, j_low_box_0, k_low_box_0(, gpu_ID_box_0 if GPU run), hostname_box_0,
+#      cost_box_1, proc_box_1, lev_box_1, i_low_box_1, j_low_box_1, k_low_box_1(, gpu_ID_box_1 if GPU run), hostname_box_1,
 #      ...
-#      cost_box_n, proc_box_n, lev_box_n, i_low_box_n, j_low_box_n, k_low_box_n]
+#      cost_box_n, proc_box_n, lev_box_n, i_low_box_n, j_low_box_n, k_low_box_n(, gpu_ID_box_n if GPU run), hostname_box_n]
 
 # Function to get efficiency at an iteration i
 def get_efficiency(i):
     # First get the unique ranks
-    costs, ranks = data[i,0::6], data[i,1::6].astype(int)
+    # The spacing of '7' is because we know this is not a GPU run; in a GPU,
+    # this should be '8' because the gpu_ID also is saved for each box
```

Comment:
In order to simplify the above, would it make sense to switch the position of `hostname` and `gpu_ID` (i.e. hostname would always be `7`, whether or not we run on GPU, and then `gpu_ID` would be `8` - if it does exist)

Reply:
I have made a change in the analysis script so it would work with a GPU or CPU run (compute the number of datafields saved per box).  I went with this alternate simplification out of a slight preference to keep the output of float data all together, then output the string data, hopefully this OK(?).